### PR TITLE
DOC-385 FB CAPI Actions

### PR DESCRIPTION
**Pull request by markzegarelli**
### Proposed changes
This will be in Private Beta, deploy on 12/2.

### CODE REVIEW

**Comment by kdaswani** [comment]
Hi @markzegarelli before we merge this, can we make a quick swap on the image used to show the `externalId` coalesce function? This is the updated image (the coalesce looks at the top-level `userId` and `anonymousId` not `properties.etc...`). I believe it is used twice. The other image that shows all of the User Data mapping is so small I think it's fine that it still says `properties.`.
Screen Shot 2021-11-30 at 9 03 08 PM

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+### Send events from both the browser and the server
+
+This approach provides a redundancy that ensures maximum signal reliability. Events that previously could have been lost (for several different reasons) when sent from the browser to Facebook Pixel, are also captured using the Facebook Conversions API. You can use this if you do not want to miss any events coming from the browser. 
+
+#### Match rate considerations
+
+For this option to work best, pass the same External ID from the browser and the server. To do this, go to your Facebook Pixel destination settings in Segment and enable the **Enable Advanced Matching** setting. Next, enable the **Use User ID or Anonymous ID as External ID** setting, or choose a custom trait for External ID using the **Advanced Match Trait Key for External ID** setting.
+
+With the Facebook Conversions API (Actions) destination, you can choose any field in your event to map to External ID. By default, Segment uses the userId (or anonymousId if not present) to set the External ID, but you can change this in the User Data object mapping of your Action.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:71]:
```suggestion
With the Facebook Conversions API (Actions) destination, you can choose any field in your event to map to External ID. By default, Segment uses the userId (or anonymousId if userId is not present) to set the External ID, but you can change this in the User Data object mapping of your Action.
```

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ hide-boilerplate: true
 hide-dossier: true
 ---
 
-Facebook Conversions API (Actions) enabled advertisers to send events from their servers directly to Facebook. Server-side events link to Facebook Pixel events, and process like browser pixel events. This means that serve-side events are used in measurement, reporting, optimization just like browser pixel events.
+Facebook Conversions API (Actions) enabled advertisers to send events from their servers directly to Facebook. Server-side events link to Facebook Pixel events, and process like browser pixel events. This means that serve-side events are used in measurement, reporting, and optimization, just like browser pixel events.
```
Comment (kdaswani) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:8]:
```suggestion
Facebook Conversions API (Actions) enables advertisers to send events from their servers directly to Facebook. Server-side events link to Facebook Pixel events, and process like browser pixel events. This means that serve-side events are used in measurement, reporting, and optimization, just like browser pixel events.
```

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+## Benefits of Facebook Conversions API (Actions) vs Facebook Conversions API Classic
+
+Segment’s Facebook Conversions API (Actions) destination provides the following benefits over the classic Facebook Conversions API destination:
+- **Fewer settings**. Data mapping for actions-based destinations happens during configuration, which eliminates the need for most settings.
+- **Clearer mapping of data**. Actions-based destinations enable you to define the mapping between the data Segment receives from your source, and the data Segment sends to Facebook Conversions API.
+- **Support for page calls**. Page calls can be sent to Facebook as a standard Page View.
+- **Support for identify calls**. Identify calls can be sent to Facebook as a standard or custom event.
+- **Support for multi-product arrays**. Products nested within arrays, like the products array in the Order Completed event, can be sent to Facebook.
```
Comment (kdaswani) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:23]:
```suggestion
- **Support for multi-product arrays**. Product data nested within arrays, like the products array in the Order Completed event, can be sent to Facebook.
```

File: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+#### Match rate considerations
+
+For this option to work best, the same External ID needs to be passed from the browser and the server. To achieve this, go to your Facebook Pixel destination settings in Segment and enable the **Enable Advanced Matching** setting. Next, enable the **Use User ID or Anonymous ID as External ID** setting, or choose a custom trait for External ID using the **Advanced Match Trait Key for External ID** setting.
+
+With the Facebook Conversions API (Actions) destination, you can choose any field in your event to map to External ID. By default, Segment uses the userId (or anonymousId if userId is not present) to set the External ID, but you can change this in the User Data object mapping of your Action.
+
```
Comment (kdaswani) [inline src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:96]: I think this paragraph was removed. Was that intentional?

> You can send additional User Data to increase the match rate for events from a server source. Collect other fields from the browser, like User Agent, IP Address, and [Facebook's cookie parameters (fbp, fbc)](https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/fbp-and-fbc){:target="_blank"}, pass them to the server, and map them in the User Data object.

Comment (markzegarelli) [reply src/connections/destinations/catalog/actions-facebook-conversions-api/index.md:96]: Hmm, not sure what happened there, will re-add.

**Review by forstisabella — APPROVED** [review]

**Pull request merged**